docs/connect-pmm.md

```diff
@@ -39,23 +39,29 @@ To connect your PMM server to Percona Platform, generate a personal access token
 
 1. Log in to PMM and go to **PMM > Settings > Percona Platform** tab to fill in the **Connect PMM to Percona Platform** form.
 
-![Connect PMM to Percona Platform](images/pmm-connect-to-platform.png)
+   ![Connect PMM to Percona Platform](images/pmm-connect-to-platform.png)
 
 2. The **PMM server ID** field is automatically populated with the ID identified for your PMM instance. Enter the name of your PMM instance and click **Get token** to go to Percona Platform Portal and generate your access token.
 3. Log into Percona Platform using your Percona Account (if you don't have an active current session).
 4. On the Profile Settings page, copy the code from the **Percona Platform Access Token** field.
    For security reasons, the access token expires after 30 minutes. Make sure to paste the code in PMM before that, or generate a new token if it expires.
 5. Back into PMM, paste the Access Token into the **Percona Platform Access Token** field and click **Connect**.
 
-To confirm that you have successfully connected the server and check the list of all servers currently connected to an organization, go to [Percona Platform](https://portal.percona.com) > **Dashboard** tab and click **View Instances** next to the **Connect your PMM** step.
+To confirm that you have successfully connected the server and check the list of all servers currently connected to an organization, go to [Percona Platform](https://portal.percona.com) **> PMM Instances**. 
 
 ### Disconnect a PMM instance
 
 Disconnect a PMM instance when you want to unlink it from your Percona Platform organization or stop monitoring it there.
 
 To disconnect a PMM server, go to > <i class="uil uil-cog"></i> **Configuration > Settings > Percona Platform** and click **Disconnect**.
 
-To confirm that the server disconnected successfully, go to Percona Platform > **Dashboard** tab > **View instances**. This displays the **PMM instances** page where you can check the list of servers currently connected to your Platform organization.
+To confirm that the server disconnected successfully, go to Percona Platform > **View instances**. This displays the **PMM instances** page where you can check the list of servers currently connected to your Platform organization.
```

Comment:
Is it Percona Platform > View instances or Percona Platform > PMM instances? If this refers to the option on the Portal left hand side menu, then it is PMM instances, please double-check